Evernote.Context/AppDataContext.cs

```diff
@@ -1,11 +1,52 @@
-﻿using Microsoft.EntityFrameworkCore;
+﻿using Evernote.Entities;
+using Microsoft.EntityFrameworkCore;
 using Microsoft.EntityFrameworkCore.Metadata.Conventions;
 
 namespace Evernote.Context {
     public class AppDataContext : DbContext {
 
         protected override void OnModelCreating(ModelBuilder modelBuilder) {
             base.OnModelCreating(modelBuilder);
+
+            modelBuilder.Entity<Note>(
+                    note => {
+
+
+                        note.HasOne(n => n.User)
```

Comment:
I would suggest using ApplyConfiguration and Configs in a separate file.
This is just styling stuff - no need to change